libs/zephyr-agent/src/lib/errors/codes.ts

```diff
@@ -419,12 +419,32 @@ Please check your network connection and try again.
   ERR_RESOLVE_REMOTES: {
     id: '001',
     message: `
-Please build {{ appUid }} with Zephyr first or add as Unmanaged applications.
-
-Note: you can read application uid as follows:
-- {{ appName }} - project.json 'name' field of remote application
-- {{ projectName }} - git repository name
-- {{ orgName }} - git organization name
+Failed to resolve remote dependency: {{ appUid }} version {{ version }}
+
+WHY THIS FAILED:
+- The remote application '{{ appName }}' has not been built with Zephyr yet
+- The specified version '{{ version }}' does not exist
+- You don't have access to this application
+- The application exists but no environment has been created
```

Comment:
Can you phrase this in a way that it can be understood that one of these is the reason and not all of them?

Reply:
Maybe better like this ?